lib/json_schema_form/umm_json_form.rb

```diff
@@ -152,7 +152,10 @@ def convert_key_to_type(value, type)
     return (value.casecmp('true') >= 0 ? true : false) if type == 'boolean'
 
     # Numbers
-    return UmmUtilities.convert_to_integer(value) if type == 'number'
+    return UmmUtilities.convert_to_number(value) if type == 'number'
+
+    # Integers
+    return UmmUtilities.convert_to_integer(value) if type == 'integer'
```

Comment:
We weren't able to come up with a use case for this, is there something in the schema that is defined as an integer vs a number?